.github/workflows/release.yml

```
@@ -0,0 +1,13 @@
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```
or change main branch to master ? :)